lerobot/configs/policy/vla.yaml

```diff
@@ -1,14 +1,6 @@
-# @package _global_
```

Comment:
Adding this line helped avoid explicitly using the name attribute (e.g., `policy.name = vla `or `env.name = aloha`), which was causing various issues with `override_dataset_stats`. The exact reason for these issues is still unclear to me. If you have any idea why this happens, I’d be happy to learn more.

Reply:
Not sure, but when you add this line you make the parameters global and hence you can access them directly. For example policy.n_obs_steps instead of e.g. policy.vla.policy.n_obs_steps